sentry-kotlin-extensions/src/test/java/io/sentry/kotlin/SentryContextTest.kt

```diff
@@ -38,11 +39,40 @@ class SentryContextTest {
             Sentry.setTag("c2", "c2value")
             assertEquals("c2value", getTag("c2"))
             assertEquals("parentValue", getTag("parent"))
-            assertNull(getTag("c1"))
+            assertNotNull(getTag("c1"))
```

Comment:
yes, that's the way it's meant to work. nice!

Reply:
We should discuss how this should work in global hub mode. Since `Sentry.getCurrentScopes` always returns `rootScopes` (think `mainHub`) it behaves differently.

Reply:
Adding a comment for further discussion on a follow up PR.